Pokedex_iosTests/Pokedex_iosTests.swift

```diff
@@ -8,7 +8,27 @@
 import XCTest
 @testable import Pokedex_ios
 
+let data = PokemonRaw(id: 4, name: "some", generation: Generation(id: 4, name: "gen1"), pokemonDetails: [PokemonDetail(name: "detail1", height: 5, weight: 5, types: [TypeElement(type: Generation(id: 3, name: "gen1"))])])
+
+
+class MockPokedexService: PokedexServiceProtocol {
+    var getPokemonsGotCalled = false
+    func getPokemons(completion: @escaping (Bool, [PokemonRaw]?, String?) -> ()) {
+        completion(true, [data], nil)
+        getPokemonsGotCalled = true
+    }
+}
+
 class Pokedex_iosTests: XCTestCase {
+    let mockTest = MockPokedexService()
+    let viewModel = PokemonListViewModel(pokedexService: MockPokedexService())
+    
+    func testGetPokemonsFromNetwork() {
+        viewModel.getPokemons()
+        XCTAssertNotEqual(viewModel.pokemons.count, 0)
```

Comment:
id better to assert the length is equal to the exact amount you passed in your mock data, instead of != 0. Is too ambiguous